packages/network-controller/src/NetworkController.ts

```diff
@@ -400,35 +400,40 @@ export class NetworkController extends BaseControllerV2<
     this.refreshNetwork();
   }
 
-  getEIP1559Compatibility() {
+  private getLatestBlock(): Promise<Block> {
```

Comment:
Ah sorry, by private I meant using the new syntax, e.g.

```suggestion
  #getLatestBlock(): Promise<Block> {
```

This syntax is preferred over `private` in all cases, as it makes the method truly private even at runtime. It's set as preferred in the latest version of our lint config as well.